src/content/docs/tutorial-kubernetes-learn/tutorial-k8s-intro.mdx

```diff
@@ -0,0 +1,148 @@
+---
+title: Understand your kubernetes system with New Relic
+---
+
+import tutorialNavigator from 'images/kubernetes_screenshot-full_navigator-main.webp'
+
+import tutorialInstall from 'images/kubernetes_screenshot-full_guidedInstall.webp'
+
+import k8sNavigatorAnimation from 'images/kubernetes_screenshot-full_navigator-animated.gif'
+
+import apmApmCompleteSummaryPage from 'images/apm_screenshot-full_apm-complete-summary-page-.webp'
+
+import infraComplete from 'images/infrastructure_screenshot-full_infra-main.webp'
+
+
+A Kubernetes system is inherently complex and Kubernetes as a whole has a reputation as intricate and challenging to grasp. Understanding Kubernetes as a concept is a complex task by itself and wanting to understand an indivdual Kubernetes system only adds more complextiy. How do you troubleshoot ephemeral containers that spin up and down before you can even access them? How do you understand the health of your system as a whole if you have hundreds of containers orchestrated at a time?
```

Comment:
"A Kubernetes system is inherently complex and Kubernetes as a whole has a reputation as intricate and challenging to grasp. Understanding Kubernetes as a concept is a complex task by itself and wanting to understand an indivdual Kubernetes system only adds more complextiy."

It seems to me this is too many words to express that Kubernetes is technically complex and difficult to understand. For this to feel like a lifeline, I think the language should be short, crisp, and direct as possible.

Reply:
I think the detail in the questions is great. Is there a third question you can add so it doesn't feel like there are only two reasons why Kubernetes is hard?

Reply:
Also, I think there's a crucial thing missing: I think it's important to state that New Relic AND this tutorial help bridge this gap between failing at Kubernetes and succeeding with it.

So, to my mind, this intro should make three main points as clear and direct as possible:

1. Kubernetes is hard (self-evident to anyone who's used it, I think)
2. Here's why it's hard (specific examples to prove you know what you're writing about!)
3. Here's how New Relic and this tutorial will help YOU, reader

Reply:
Agree with paperclypse, I like the idea behind it but I think it can be cleaned up a bit. It feels a bit repetitive as it stands now.